### ISSUE TITLE
Feature-Request: Please add preferred language (for final output)

### ISSUE THREAD

**Issue by MeinDeutschkurs** [ItzCrazyKns/Perplexica#296]
Independently from the input language Perplexica always replies in English. 

Could you please introduce "preferred output language" in the settings? With prompting like "Please write your final output in {final_output_language}." the preferred language could be used. Thx for considering this.

@ItzCrazyKns: I LOVE THIS PROJECT! IT'S AMAZING!!

**Comment by AlexTracks** [comment]
Yeah, that would be great.

I haven't tested it yet, but can't you specify that in the prompt?

**Comment by ItzCrazyKns** [comment]
The issue is same as this one https://github.com/ItzCrazyKns/Perplexica/issues/72, so I will close it, you can follow the other issue for updates.

**Comment by MeinDeutschkurs** [comment]
No, this is not the same. It has to do with the wish to get a new option in the settings. Preferred output language. I cannot see, where to set the preferred output language, because I have no field to set additional prompting exclusively to the final output prompt.

**Comment by PeterDaveHello** [comment]
@ItzCrazyKns I think this request is a bit different from #72. That issue is about system prompts for various tasks, but the preferred output language is a separate thing. Adding a setting for this would be super user-friendly and helpful, as we could support many common languages directly.

Thanks for considering it!
